include/abb_librws/rws_interface.h

```diff
@@ -63,6 +63,16 @@ class RWSInterface
      * \brief The system's name.
      */
     std::string system_name;
+
+    /**
+     * \brief The system's type.
```

Comment:
Perhaps add an example of what a "type" is of a "system"?

Reply:
Clarified in 90ab7a9.

The type can for example be `Virtual Controller`.